src/instructlab/model/download.py

```diff
@@ -264,7 +265,13 @@ def download(self):
 )
 @click.option(
     "--release",
-    default="main",  # TODO: add to config.yaml
+    "-rl",
+    "releases",
+    multiple=True,
+    default=[
+        "main",
+        "main",
```

Comment:
do we need 'main' twice here?

Reply:
we need 2 release entries for the 2 default models we are downloading by default now
there are better ways to do this but this seemed like the most straightforward for what is being pursued, anything more felt like going out of the way to make it more complicated